NetworkDebugLog.swift

```diff
@@ -13,15 +13,19 @@ class NetworkDebugLog {
     ///
     /// - Parameters:
     ///   - request: An object that provides request metadata
-    static func log(with request: URLRequest) {
-        #if DEBUG
+    ///   - displayNetworkDebugLog: An object that provides a location for displaying logs
+    static func log(with request: URLRequest, displayNetworkDebugLog: DisplayNetworkDebugLog?) {
+        switch displayNetworkDebugLog {
+        case .console:
             debugPrint()
             debugPrint("==>>==========================================")
```

Comment:
I think this characters structure should be moved to constant, because this separator lines uses at least 4 times.